operating/build-time-network-policy-tools.adoc

```diff
@@ -67,7 +176,106 @@ include::modules/roxctl-netpol-connectivity-map-command-options.adoc[leveloffset
++
+For more information about optional parameters, see xref:../operating/build-time-network-policy-tools.adoc#roxctl-netpol-connectivity-diff-command-options_network-policy-tools[roxctl netpol connectivity diff command options].
++
+[NOTE]
+====
+The command considers all YAML files that you can accept using `kubectl apply -f`, and then these become valid inputs for your `roxctl netpol connectivity diff` command.
+====
++
+.Example output
++
+====
+[%autowidth,options="header"]
+|===
+
+|diff-type |source |destination |dir 1 |dir 2 |workloads-diff-info
+
+|changed
+|default/frontend[Deployment]
+|default/backend[Deployment]
+|TCP 9090
+|TCP 9090,UDP 53
+|
+
+|added
+|0.0.0.0-255.255.255.255
+|default/backend[Deployment]
+|No Connections
+|TCP 9090
+|
+
+|===
+====
+
+The semantic difference report gives you an overview of the connections that were changed, added, or removed in `dir2` compared to the connections allowed in `dir1`. When you review the output, each line represents one allowed connection that was added, removed, or changed in `dir2` compared to `dir1`.
+
+The following are example outputs generated by the `roxctl netpol connectivity diff` command in various formats:
+
+** link:https://github.com/np-guard/netpol-analyzer/blob/main/tests/netpol-diff-example-minimal/diff_output_from_netpol-analysis-example-minimal.txt[Example 1: text format]
++
+** link:https://github.com/np-guard/netpol-analyzer/blob/main/tests/netpol-diff-example-minimal/diff_output_from_netpol-analysis-example-minimal.md[Example 2: md format]
++
+** link:https://github.com/np-guard/netpol-analyzer/blob/main/tests/netpol-diff-example-minimal/diff_output_from_netpol-analysis-example-minimal.csv[Example 3: csv format]
+
+If applicable, the `workloads-diff-info` provides additional details about added or removed workloads related to the added or removed connection.
+
+.Example output
+
+====
+[%autowidth,options="header"]
+|===
+
+|diff-type |source |destination |dir 1 |dir 2 |workloads-diff-info
+
+|added
+|payments/gateway[Deployment]
+|payments/visa-processor-v2[Deployment]
+|No Connections
+|TCP 8080
+|workload payments/visa-processor-v2[Deployment] added
+
+|added
+|{ingress-controller}
+|frontend/blog[Deployment]
+|No Connections
+|TCP 8080
+|workload frontend/blog[Deployment] added
+
+|added
+|{ingress-controller}
+|zeroday/zeroday[Deployment]
+|No Connections
+|TCP 8080
+|workload zeroday/zeroday[Deployment] added
+
+|===
+====
+
+In this example, the first row indicates that `payments/gateway[Deployment]` can now connect to `payments/visa-processor-v2[Deployment]` on TCP 8080. The second and third rows indicate similar added connections involving an ingress controller and their respective destinations. The `workloads-diff-info` column contains information about the affected workloads.
```

Comment:
I would remove this example output and the lines below it (it is unclear from which example / resources it is produced from). 
Instead I would replace it with following additional details:

If a connection from workload `A` to workload `B` is removed since workload `B` was removed, the `workloads-diff-info` will add that information (that the this workload was removed). But if such a connection was removed only due to change in network policies, and neither `A` nor `B` were removed, then the workloads-diff-info will be empty.